apps/implement-use-case.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: page
+key: apps-implement-use-case
+title: Implementing a use case
+---
+
+## Image Service
+
+To add a new image to a product, this is what you would need to do:
+
+1. Use the [`POST` products/slideshow](page:apps-api-post-shops-shopid-products-productid-slideshow-information) endpoint and post a new image to the slideshow of the product.
+If you want to replace an existing image, delete that image first using the [`DELETE` products/slideshow{imageName}](page:apps-api-delete-shops-shopid-products-productid-slideshow-imagename-information) endpoint.
+
+2. After having added a new image to the slideshow, you can assign the image to a product using the [`PATCH` products](page:apps-api-patch-shops-shopid-products-productid-information) endpoint.
+Update the product image by addressing the attribute path `productImage`.
+
+Example:
```

Comment:
Leider nicht, habe ich schon in allen möglichen Varianten probiert. Damit müssen wir so leben.